sdk/protocol2.go

```diff
@@ -132,6 +132,19 @@ func (d *EntityData) NewMetricSet(eventType string) metric.MetricSet {
 	return ms
 }
 
+// Event method creates or retrieves an already created Event.
+func (d *EntityData) Event(summary, category string) *Event {
```

Comment:
Do we need to return the reference to the event? I would say that once you have added the event to the entity data, you shouldn't work with that event anymore.

If you agree with my comments, then probably would make sense to rename the method to something like `AddEvent` to make it more explicit.

Reply:
Yeah, I was thinking about it. But you're right. We should not work with this event later. 

However if the event already exists, what should we do? Fail or do nothing? Or add the same event again? Once we decide it we can propose the name.